.changeset/perfect-sheep-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/build-utils": patch
+---
+
+[build-utils] Push back `nodejs16.x` discontinue date to 2024-02-06
```

Comment:
The prefix is redundant since this will be going into build-utils own changelog file.

```suggestion
Push back `nodejs16.x` discontinue date to 2024-02-06
```

Reply:
@TooTallNate Sounds good. What about PR titles. Do you want the prefix there?

Reply:
I've been prefixing both commit messages and PR titles. Only the changeset do I omit the prefix.

Reply:
I used the button to generate the changeset so it used the PR title (or perhaps commit title) 🤷‍♂️

I wish more of this was automated 😄